src/foam/nanos/dig/DigUtil.java

```diff
@@ -0,0 +1,84 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package foam.nanos.dig;
+
+import foam.core.X;
+import foam.lib.*;
+import foam.lib.csv.CSVOutputter;
+import foam.lib.json.JSONParser;
+import foam.lib.json.Outputter;
+import foam.lib.json.OutputterMode;
+import foam.nanos.dig.exception.DigErrorMessage;
+import foam.nanos.http.Format;
+
+import javax.servlet.http.HttpServletResponse;
+import java.io.PrintWriter;
+
+public class DigUtil {
```

Comment:
Document what this class/file is for.

Reply:
Fixed in https://github.com/foam-framework/foam2/pull/2832/commits/340b4db558bd19d16930d65f2f1707e5ff057123.